src/test/java/org/jabref/logic/integrity/IntegrityCheckTest.java

```diff
@@ -368,7 +367,7 @@ void testEntryIsUnchangedAfterChecks() {
         new IntegrityCheck(context,
                 mock(FilePreferences.class),
                 createBibtexKeyPatternPreferences(),
-                           new JournalAbbreviationRepository(new Abbreviation("IEEE Software", "IEEE SW")), true, false)
+                new JournalAbbreviationRepository(), true, false)
```

Comment:
Just stumbled over this - can we get rid of `true` and `false` somehow? - enums maybe?

Reply:
I don't see anything wrong with these booleans (and it's not related to this PR anyway)